sp/include/optee_sp_internal_api.h

```diff
@@ -0,0 +1,66 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2020-2021, Arm Limited.
+ */
+
+#ifndef SP_INCLUDE_OPTEE_SP_INTERNAL_API_H_
+#define SP_INCLUDE_OPTEE_SP_INTERNAL_API_H_
+
+#include <stddef.h>
+#include <stdint.h>
+#include "compiler.h"
+#include "util.h"
+
+/*
+ * The file describes the API between the SP dev kit and the SP. It contains the
+ * function prototypes that the user code must define.
+ * The SP code base should also contain a header file named
+ * "optee_sp_user_defines.h" for passing the following definitions to the SP dev
+ * kit:
+ * * OPTEE_SP_HEAP_SIZE: Heap size in bytes
+ * * OPTEE_SP_UUID: UUID of the SP as an sp_uuid structure
+ * * OPTEE_SP_STACK_SIZE: Stack size in bytes
+ */
+
+/*
+ * The file defines the functions and variables that other
```

Comment:
something is missing in the sentence

Reply:
This is some leftover comment which I've now removed. Lines 14-23 explain the purpose of this file properly.